common/src/main/kotlin/entity/DiscordGuild.kt

```diff
@@ -81,82 +81,54 @@ data class DiscordGuild(
     val unavailable: OptionalBoolean = OptionalBoolean.Missing,
-    @SerialName("member_count")
-    val memberCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("voice_states")
-    val voiceStates: Optional<List<DiscordVoiceState>> = Optional.Missing(),
+    @SerialName("member_count") val memberCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("voice_states") val voiceStates: Optional<List<DiscordVoiceState>> = Optional.Missing(),
     val members: Optional<List<DiscordGuildMember>> = Optional.Missing(),
     val channels: Optional<List<DiscordChannel>> = Optional.Missing(),
     val threads: Optional<List<DiscordChannel>> = Optional.Missing(),
     val presences: Optional<List<DiscordPresenceUpdate>> = Optional.Missing(),
-    @SerialName("max_presences")
-    val maxPresences: OptionalInt? = OptionalInt.Missing,
-    @SerialName("max_members")
-    val maxMembers: OptionalInt = OptionalInt.Missing,
-    @SerialName("vanity_url_code")
-    val vanityUrlCode: String?,
+    @SerialName("max_presences") val maxPresences: OptionalInt? = OptionalInt.Missing,
+    @SerialName("max_members") val maxMembers: OptionalInt = OptionalInt.Missing,
+    @SerialName("vanity_url_code") val vanityUrlCode: String?,
     val description: String?,
     val banner: String?,
-    @SerialName("premium_tier")
-    val premiumTier: PremiumTier,
-    @SerialName("premium_subscription_count")
-    val premiumSubscriptionCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("preferred_locale")
-    val preferredLocale: String,
-    @SerialName("public_updates_channel_id")
-    val publicUpdatesChannelId: Snowflake?,
-    @SerialName("max_video_channel_users")
-    val maxVideoChannelUsers: OptionalInt = OptionalInt.Missing,
-    @SerialName("approximate_member_count")
-    val approximateMemberCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("approximate_presence_count")
-    val approximatePresenceCount: OptionalInt = OptionalInt.Missing,
-    @SerialName("welcome_screen")
-    val welcomeScreen: Optional<DiscordWelcomeScreen> = Optional.Missing(),
-    @SerialName("nsfw_level")
-    val nsfwLevel: NsfwLevel
+    @SerialName("premium_tier") val premiumTier: PremiumTier,
+    @SerialName("premium_subscription_count") val premiumSubscriptionCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("preferred_locale") val preferredLocale: String,
+    @SerialName("public_updates_channel_id") val publicUpdatesChannelId: Snowflake?,
+    @SerialName("max_video_channel_users") val maxVideoChannelUsers: OptionalInt = OptionalInt.Missing,
+    @SerialName("approximate_member_count") val approximateMemberCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("approximate_presence_count") val approximatePresenceCount: OptionalInt = OptionalInt.Missing,
+    @SerialName("welcome_screen") val welcomeScreen: Optional<DiscordWelcomeScreen> = Optional.Missing(),
+    @SerialName("nsfw_level") val nsfwLevel: NsfwLevel
```

Comment:
whilst youre updating guilds, you could also add `guild_scheduled_events` and `premium_progress_bar_enabled`

Reply:
`stage_instances` and `stickers` are also missing